spec/features/applications/new_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+RSpec.describe "Applications" do
+
+  before :each do
+    @shelter_1 = Shelter.create(name: "Aurora shelter", city: "Aurora, CO", foster_program: false, rank: 9)
+    @shelter_1.pets.create(name: "Mr. Pirate", breed: "tuxedo shorthair", age: 5, adoptable: false)
+    @shelter_1.pets.create(name: "Clawdia", breed: "shorthair", age: 3, adoptable: true)
+    @shelter_1.pets.create(name: "Ann", breed: "ragdoll", age: 5, adoptable: true)
+  end
+
+  describe 'Starting an Application' do
+    it 'has a link on pet index page' do
+      visit "/pets"
+      
+      expect(page).to have_link "Start an Application", href: "/applications/new"
```

Comment:
I think we should probably add click_link "Start an Application" and then an expect(current_path_.to eq('applications/new'), but I'm not sure. Might be a bit redundant, but it'll tell us if the path gets broken for some reason.